.circleci/config.yml

```diff
@@ -1,29 +0,0 @@
-version: 2
-jobs:
- build:
```

Comment:
Someone needs to actually remove the build configuration from CircleCI. I tried but I apparently have no power there.